requests_cache/core.py

```diff
@@ -259,12 +264,18 @@ def clear():
 def _patch_session_factory(session_factory=CachedSession):
     requests.Session = requests.sessions.Session = session_factory
 
-
-def _normalize_parameters(params):
+def _normalize_parameters(params, ignored_parameters=None):
     """ If builtin dict is passed as parameter, returns sorted list
     of key-value pairs
     """
-
     if type(params) is dict:
-        return sorted(params.items(), key=itemgetter(0))
+        params = sorted(params.items(), key=itemgetter(0))
+
+    if ignored_parameters:
```

Comment:
If custom dict, like ``OrderedDict``, is passed, ``type(params) is dict`` is ``False`` (it's intentional) and ``for k, v in params`` will fail. So, please add check here:
``` python
if isinstance(params, collections.Mapping):
   params = params.items()
```